src/scripts/importMetaData.ts

```diff
@@ -0,0 +1,73 @@
+import * as Figma from 'figma-js';
+import { promises as fs } from 'fs';
+import startCase from 'lodash.startcase';
+import path from 'path';
+
+if (!process.env.FIGMA_FILENAME || !process.env.FIGMA_PERSONAL_ACCESS_TOKEN) {
```

Comment:
I can't figure out how to get the ENV vars working here. This script works by setting the env vars into constants manually.

Reply:
I figured it out. Combination of missing the dotenv-flow config() and a misnamed env var that took me 30 mins to find…